jgrapht-core/src/test/java/org/jgrapht/alg/decompostion/NiceDecompositionBuilderTestUtil.java

```diff
@@ -0,0 +1,115 @@
+package org.jgrapht.alg.decompostion;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.*;
+import java.util.Map.*;
+
+import org.jgrapht.*;
+import org.jgrapht.graph.*;
+
+public final class NiceDecompositionBuilderTestUtil
+{
+    //static class
+    private NiceDecompositionBuilderTestUtil() {}
+    
+    /**
+     * tests whether the tree decomposition (decomposition, map) with root is nice
```

Comment:
Let's rephrase this: **T**ests ... whether the tree decomposition, consisting of the decomposition and the map with the given root is nice. Or something similar.